eip/EIPS/eip-4337.md

```diff
@@ -58,45 +69,88 @@ The core interface of the entry point contract is as follows:
 
 ```c++
 function handleOps
-    (UserOperation[] calldata ops, address payable beneficiary)
-    public;
+    (UserOperation[] calldata ops, address payable beneficiary
+     AggregatorInfo[] calldata aggregators, uint[] memory execOrder
+    ) public
 
 function simulateValidation
-    (UserOperation calldata userOp)
-    external returns (uint256 preOpGas, uint256 prefund) {
+    (UserOperation calldata userOp, aggregatedSignature)
```

Comment:
Also `aggregatedSignature` here is also an individual signature, not aggregated signature. It is not possible to check a single element of an aggregated signature as far as I understand.

Reply:
Well, at least for BLS, as far as I understand, signature for array of 1 is in fact an individual signature.  I don't like calling this parameter "aggregatedSignature" if it is known to be a signature of an array of 1.

Reply:
it is an aggregated version of all userOps signatures together into a single one.